src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt

```diff
@@ -321,30 +322,142 @@ class CreateMafVcf : CliktCommand(help = "Create gVCF and hVCF from Anchorwave M
             else lastVariant.getAttributeAsInt("ASM_Start",region.second.position)
         }
 
+        val regions = buildNewAssemblyRegions(newASMStart,newASMEnd,variants)
```

Comment:
Was the intent to remove the check for strand match on line 311 (based on the rework to use regions)?  It still fails on that line in the at-scale testing.  I'll comment that out and see how it does.